utils/install-ubuntu1804.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Installs a Swift for TensorFlow toolchain from scratch on Ubuntu 18.04.
+#
+# Usage:
+#   ./install-ubuntu1804
+#     [--toolchain-url TOOLCHAIN_URL]
+#     [--jupyter-url JUPYTER_URL]
+#     [--cuda CUDA_VERSION]
+#     [--no-jupyter]
+#     [--install-location INSTALL_LOCATION]
+#
+# Arguments:
+#   --toolchain-url: Specifies the URL for the toolchain. Defaults to the latest
+#                    nightly CPU-only toolchain.
+#   --jupyter-url: Specifies the URL for swift-jupyter. Defaults to the latest
+#                  nightly build. Set this to the empty string to disable
+#                  swift-jupyter installation.
+#   --install-location: Directory to extract the toolchain. Defaults to
+#                       "./swift-toolchain".
+
+set -exuo pipefail
+
+TOOLCHAIN_URL=https://storage.googleapis.com/swift-tensorflow-artifacts/nightlies/latest/swift-tensorflow-DEVELOPMENT-ubuntu18.04.tar.gz
```

Comment:
Is this available for ubuntu 16.04 too?

Reply:
No, our automated builds only do 18.04. The full list of urls for all the versions we publish is here: https://github.com/tensorflow/swift/blob/master/Installation.md